scout/server/blueprints/variants/templates/variants/str-variants.html

```diff
@@ -121,6 +121,13 @@
 <div class="container-fluid">
   {{ pagination_footer(more_variants, page) }}
 </div>
+<!--  Pedigree_panel -->
+<div class="row">
```

Comment:
You tell me what is best practice, but I probably would have kept the row div within the if statement for now?

Reply:
Totally agree. Thanks